lib/eventify_pro/client.rb

```diff
@@ -44,6 +48,13 @@ def process_error(error)
       raise Error, 'Could not publish event'
     end
 
+    def log_error(message, method, params:)
+      message = "[EVENTIFY_PRO] Message: #{message}\nMethod: #{method} with params: #{params}" # rubocop:disable LineLength
+      logger.info(message) if logger.respond_to?(:info)
+
+      logger.info(message)
```

Comment:
Good point. Since it seems like logger will always respond to info now, should there not be an if statement? Is that what you are saying?